pkg/types/types.go

```diff
@@ -33,7 +33,6 @@ type RelayArgs struct {
 }
 
 type Relayer interface {
-	Service
```

Comment:
None of the Relayer implementations actually use the Service methods for anything interesting, so this is just requiring unnecessary no-op boilerplate.